ROADMAP.md

```diff
@@ -31,6 +29,12 @@
 - [n-gram](https://github.com/machinetranslate/machinetranslate.org/issues/108)
 - [Vector](https://github.com/machinetranslate/machinetranslate.org/issues/112)
 - [Language model](https://github.com/machinetranslate/machinetranslate.org/issues/107)
+- Word embeddings
+
+### Concepts - Translator-Machine interaction
```

Comment:
I'd just call it 

> ### Concepts - Human translation

(since for any x here, it's considered to be the intersection of x and machine translation)